src/LearnositySdk/Utils/Json.php

```diff
@@ -71,6 +74,10 @@ function ($match) {
             $result = str_replace('\/', '/', $result);
         }
 
+        foreach($highPrecisionFloatMap as $entry) {
```

Comment:
I wonder if you could just use an actual map : `[ scientificValueString => floatValueString ]`.